suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -560,14 +555,27 @@ export const createDescriptorPreloadedDiscoveryThunk = createThunk(
             return;
         }
 
-        const networksSymbols = networks.map(network => network.symbol);
+        // Some cardano derivation may not be supported by the device. We need to exclude them from total count.
+        const isAvailableAccount = (networkType: NetworkType, accountType: AccountType) =>
+            networkType !== 'cardano' ||
+            (availableCardanoDerivations as string[]).includes(accountType);
 
-        const discoveryNetworksTotalCount = networksSymbols.length;
+        const discoveryNetworksTotalCount = networks.reduce((count, network) => {
+            count += isAvailableAccount(network.networkType, NORMAL_ACCOUNT_TYPE) ? 1 : 0;
```

Comment:
I could have written `count += Number(isAvailableAccount..);` instead of ternary, but this is clearer :smile: